firebase-perf/src/main/java/com/google/firebase/perf/config/RemoteConfigManager.java

```diff
@@ -69,14 +74,25 @@ private RemoteConfigManager() {
         );
   }
 
-  @VisibleForTesting
   RemoteConfigManager(Executor executor, FirebaseRemoteConfig firebaseRemoteConfig) {
+    this(
+        executor,
+        firebaseRemoteConfig,
+        MIN_APP_START_CONFIG_FETCH_DELAY_MS
+            + new Random().nextInt(RANDOM_APP_START_CONFIG_FETCH_DELAY_MS));
+  }
+
+  @VisibleForTesting
+  RemoteConfigManager(
+      Executor executor, FirebaseRemoteConfig firebaseRemoteConfig, long fetchDelay) {
```

Comment:
Lets call the new variable "appStartConfigFetchDelayInMs". Basically, we are looking at a delay from app start before starting to fetch. Lets have a name to represent that. The current name includes a notion that we are requesting a delay when this instance is created (This instance could get created much later than app start).

Reply:
done